src/nfts/NftsLoadError.tsx

```diff
@@ -27,7 +31,7 @@ export default function NftsLoadError() {
   }, [])
 
   return (
-    <SafeAreaView style={styles.safeArea} testID="NftsInfoCarousel/NftsLoadErrorScreen">
+    <SafeAreaView style={styles.safeArea} testID={testID ?? 'NftsInfoCarousel/NftsLoadErrorScreen'}>
```

Comment:
nit: if this component is reusable, i don't think it makes sense to have a default testID? i guess it's not very consequential here but it does feel like a trap for forgetting to pass this prop in and having the wrong testID assigned

Reply:
Fixed in c85222bf3c67d3e2afbb5d5df8f3cabeead68fe0!